web/profiles/custom/os2loop/modules/os2loop_upvote/src/Helper/FlagHelper.php

```diff
@@ -46,15 +62,24 @@ public function preprocessFlag(array &$variables) {
   /**
    * Add number of upvotes to flag.
    */
-  public function flagCounts(array &$variables) {
+  public function preprocessFlag(array &$variables) {
     $comment = $variables['flaggable'];
-    $flag_service = \Drupal::service('flag.count');
-    $flag_counts = $flag_service->getEntityFlagCounts($comment);
+    $flag_counts = $this->flagService->getEntityFlagCounts($comment);
     if (isset($flag_counts['os2loop_upvote_upvote_button'])) {
       $variables['upvotes'] = intval($flag_counts['os2loop_upvote_upvote_button']);
     }
     // Clear cache, so the UI reflects changes in top comment.
     Cache::invalidateTags($comment->getCacheTags());
   }
 
+  /**
+   * Add styling class to top comment.
+   */
+  public function preprocessComment(array &$variables) {
```

Comment:
Consider adding `Implements hook_preprocess_comment()` at the top of the comment to make it clear that this function actually does that.